src/System.Data.Common/dir.props

```diff
@@ -5,7 +5,7 @@
     <AssemblyVersion>4.2.0.0</AssemblyVersion>
     <AssemblyKey>MSFT</AssemblyKey>
     <IsNETCoreApp>true</IsNETCoreApp>
-    <IsDesktopFacade>true</IsDesktopFacade>
     <IsUAP>true</IsUAP>
+    <IsNetFxNETStandard>true</IsNetFxNETStandard>
```

Comment:
So a follow up is how come the baseline is not emptied out yet? Does that mean that we are still missing a few assemblies with netfx configurations?